package.json

```diff
@@ -44,6 +44,7 @@
     "openzeppelin-solidity": "^2.4.0",
     "sol-merger": "^2.0.1",
     "synthetix-js": "^2.21.6",
+    "tmp-promise": "^2.0.2",
```

Comment:
This is only used in unit tests and should thus be in `dev-dependencies`. In fact, there are probably many items here that are only `dev-dependencies` which should be moved (e.g. `eth-gas-reporter`, `eslint`, ...)